extensions/common/xwalk_extension_server.cc

```diff
@@ -360,12 +360,11 @@ void XWalkExtensionServer::Invalidate() {
 
 namespace {
 base::FilePath::StringType GetNativeLibraryPattern() {
-  const base::string16 library_pattern = base::GetNativeLibraryName(
-      base::UTF8ToUTF16("*"));
+  const std::string library_pattern = base::GetNativeLibraryName("*");
```

Comment:
The upstream patch you have referenced above has string16 but looks like there was another [change](https://codereview.chromium.org/2048523002) made upstream later which has std::string. Anyway, this looks OK to me.